README.md

```diff
@@ -12,6 +12,19 @@ A web application for providing access to Digital Objects in SDR Storage.
   + oracle - integration, staging, production
 
 ## Getting Started
+
+- `git clone` and `cd` into the cloned repository
+- setup mysql
+
+  ```sh
+  echo "For db creation, enter the 'root' user password:"
+  mysql --user=root -p < db/mysql_db_create.sql
```

Comment:
Would you please comment about what each step does? e.g. (Create the sdrAdmin user and the archive_catalog_test and archive_catalog_development databases.  Create tables for the test database.  Create tables for the dev database)